src/sentry/static/sentry/app/components/events/interfaces/frame/trimPackage.tsx

```diff
@@ -0,0 +1,7 @@
+function trimPackage(pkg: string) {
+  const pieces = pkg.split(/^([a-z]:\\|\\\\)/i.test(pkg) ? '\\' : '/');
+  const filename = pieces[pieces.length - 1] || pieces[pieces.length - 2] || pkg;
+  return filename.replace(/\.(dylib|so|a|dll|exe)$/, '');
+}
+
+export default trimPackage;
```

Comment:
You have a few modules with only a single function. Would it be better to collect them into a single module and used named exports/imports?

Reply:
hmm I was thinking about that, but what would the file name be like?

Reply:
Generally, `utils`